src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.MultipleStatementsCodeGenerator.cs

```diff
@@ -84,24 +63,6 @@ protected override ImmutableArray<StatementSyntax> GetInitialStatementsForMethod
                     return list.ToImmutable();
                 }
 
-                protected override SyntaxNode GetOutermostCallSiteContainerToProcess(CancellationToken cancellationToken)
-                {
-                    var callSiteContainer = GetCallSiteContainerFromOutermostMoveInVariable(cancellationToken);
-                    if (callSiteContainer != null)
-                    {
-                        return callSiteContainer;
-                    }
```

Comment:
top half of this moved up to base type.  bottom half to the 'selection result' type.